### PR TITLE
split trainer tests

### PR DESCRIPTION
**Pull request by Borda**
## What does this PR do?
Fixes simple split trainer tests as they become too long...

## PR review    
Anyone in the community is free to review the PR once the tests have passed.     
If we didn't discuss your PR in Github issues there's a high chance it will not be merged.

## Did you have fun?
Make sure you had fun coding 🙃

### CODE REVIEW

File: tests/trainer/test_dataloaders.py [hunk 1]
```diff
@@ -0,0 +1,324 @@
...
+    trainer.test()
+
+    assert len(trainer.val_dataloaders) == 1, \
+        f'`val_dataloaders` not initiated properly, got {trainer.val_dataloaders}'
+    assert len(trainer.test_dataloaders) == 1, \
+        f'`test_dataloaders` not initiated properly, got {trainer.test_dataloaders}'
+
+
+def test_multiple_dataloaders_passed_to_fit(tmpdir):
+    """ Verify that multiple val & test dataloaders can be passed to fit """
```
Comment (Borda) [inline tests/trainer/test_dataloaders.py]:
```suggestion
    """Verify that multiple val & test dataloaders can be passed to fit."""
```

File: tests/trainer/test_dataloaders.py [hunk 2]
```diff
@@ -0,0 +1,324 @@
...
+    fit_options = dict(train_dataloader=model._dataloader(train=True),
+                       val_dataloaders=model._dataloader(train=False),
+                       test_dataloaders=model._dataloader(train=False))
+
+    results = trainer.fit(model, **fit_options)
+
+    trainer.test()
+
+    assert len(trainer.val_dataloaders) == 1, \
+        f'`val_dataloaders` not initiated properly, got {trainer.val_dataloaders}'
```
Comment (Borda) [inline tests/trainer/test_dataloaders.py]:
```suggestion
        f'"val_dataloaders` not initiated properly, got {trainer.val_dataloaders}"
```

File: tests/trainer/test_dataloaders.py [hunk 3]
```diff
@@ -0,0 +1,324 @@
...
+    assert len(trainer.test_dataloaders) == 1, \
+        f'`test_dataloaders` not initiated properly, got {trainer.test_dataloaders}'
```
Comment (Borda) [inline tests/trainer/test_dataloaders.py]:
```suggestion
        f'"test_dataloaders` not initiated properly, got {trainer.test_dataloaders}"
```

File: tests/trainer/test_dataloaders.py [hunk 4]
```diff
@@ -0,0 +1,324 @@
...
+
+    # fit model
+    trainer = Trainer(**trainer_options)
+    fit_options = dict(val_dataloaders=model._dataloader(train=False),
+                       test_dataloaders=model._dataloader(train=False))
+    _ = trainer.fit(model, **fit_options)
+    trainer.test()
+
+    assert len(trainer.val_dataloaders) == 1, \
+        f'`val_dataloaders` not initiated properly, got {trainer.val_dataloaders}'
```
Comment (Borda) [inline tests/trainer/test_dataloaders.py]:
```suggestion
        f"`val_dataloaders` not initiated properly, got {trainer.val_dataloaders}"
```

File: tests/trainer/test_dataloaders.py [hunk 5]
```diff
@@ -0,0 +1,324 @@
...
+    assert len(trainer.test_dataloaders) == 1, \
+        f'`test_dataloaders` not initiated properly, got {trainer.test_dataloaders}'
```
Comment (Borda) [inline tests/trainer/test_dataloaders.py]:
```suggestion
        f'"test_dataloaders` not initiated properly, got {trainer.test_dataloaders}"
```

File: tests/models/utils.py [hunk 6]
```diff
@@ -188,13 +188,13 @@ def run_prediction(dataloader, trained_model, dp=False, min_acc=0.50):
         acc = torch.tensor(acc)
         acc = acc.item()
 
-    assert acc >= min_acc, f'this model is expected to get > {min_acc} in test set (it got {acc})'
+    assert acc >= min_acc, f"This model is expected to get > {min_acc} in test set (it got {acc})"
```
Comment (williamFalcon) [inline tests/models/utils.py]: we should be using single quotes  everywhere

Comment (Borda) [reply tests/models/utils.py]: ok, I see it here https://www.python.org/dev/peps/pep-0498/ :]

Comment (Borda) [reply tests/models/utils.py]: even I found both usage equally frequent... https://realpython.com/python-f-strings/

**Pull request merged**